app/views/auth.html

```diff
@@ -11,11 +11,17 @@
     </div>
 
     <div class="loginFields">
+<label>
       <input type="text" ng-model="user.email" class="form-control" placeholder="Email Address">
-
-      <input type="password" ng-model="user.password" class="form-control" placeholder="Password">
-      <input type="password" ng-model="user.confirmpassword" ng-show="amRegistering"  class="form-control" ng-class="{fieldInvalid: (user.password != user.confirmpassword)}" placeholder="Confirm Password">
-  </div>
+</label>
+<label>  
+<input type="password" ng-model="user.password" class="form-control" placeholder="Password">
+</label>
+<label>      
+<input type="password" ng-model="user.confirmpassword" ng-show="amRegistering"  class="form-control" ng-
+class="{fieldInvalid: (user.password != user.confirmpassword)}" placeholder="Confirm Password">
```

Comment:
this is where ng-class is broken. Anew line was created when it should be one single line.